src/components/EditContactInformation.jsx

```diff
@@ -82,239 +142,378 @@ const EditContactInformation = () => {
+                      />
+                      <CustomAccordionItem
+                        title="Do not accept cash"
+                        panelInfo="Do not accept cash! Check should be addressed to Feeding Pets of the Homeless® and sent to headquarters."
+                      />
+                      <CustomAccordionItem
+                        title="If a donor requests a donation receipt"
+                        panelInfo="If a donor requests a donation receipt, please use form included in the startup kit and return to us. We will then send an official receipt to them from headquarters."
+                      />
+                      <AccordionItem borderColor={"white"}>
+                        <Text>
+                          <AccordionButton>
+                            <Box as="span" flex="1" textAlign={'left'}>
+                              <Text fontWeight={'700'} fontSize={'16px'}>
+                                Tips on how to increase donations
+                              </Text>
+                            </Box>
+                            <AccordionIcon />
+                          </AccordionButton>
+                        </Text>
+                        <AccordionPanel pb={4}>
+                          <Flex alignItems={'center'} pb={1}>
+                            <Icon as={BiCalendar} />
+                            <Text>
+                              Host events and ask that a donation of pet food be the admittance
+                              ticket.
+                            </Text>
+                          </Flex>
+                          <Flex alignItems={'center'} pb={1}>
+                            <Icon as={BiBone} />
+                            <Text>
+                              Offer a service with a donation as the fee for your services. Example:
+                              grooming, pet sitting, boarding, etc.
+                            </Text>
+                          </Flex>
+                          <Flex alignItems={'center'} pb={1}>
+                            <Icon as={BiCar} />
+                            <Text>Fill a truck or police vehicle (popular choice).</Text>
+                          </Flex>
+                          <Flex alignItems={'center'} pb={1}>
+                            <Icon as={BiCopyAlt} />
+                            <Text>
+                              Print our newsletter or flyer and post it to boards in your community
+                              with your business name and address.
+                            </Text>
+                          </Flex>
+                          <Flex alignItems={'center'}>
+                            <Icon as={BiConversation} />
+                            <Text>We are always open to other suggestions.</Text>
```

Comment:
for each TabPanel, make a component for the tab since this file is looking way too long